src/Symfony/Component/Security/Http/Firewall/BasicAuthenticationListener.php

```diff
@@ -80,7 +80,7 @@ public function handle(GetResponseEvent $event)
             }
 
             if (null !== $this->logger) {
-                $this->logger->info(sprintf('Authentication request failed for user "%s": %s', $username, $failed->getMessage()));
+                $this->logger->info(sprintf('Basic authentication failed for user "%s"', $username), array('exception' => $failed));
```

Comment:
you could remove the ``for user "John"`` from the message, putting the username in the context instead